src/routes/ratingcourse.route.js

```diff
@@ -0,0 +1,12 @@
+const M = require('../models');
+
+module.exports = (router) => {
+  router.post('/ratingcourse', async (req, res) => {
+    M.RatingCourse.create(req.body)
+      .then(() => res.sendStatus(200))
```

Comment:
Can you return the rating if it was correctly inserted ?

Reply:
rating returned in the request in the new commit